pkg/printer/printer.go

```diff
@@ -161,13 +160,12 @@ func (p *Printer) GetHostNames(f v1.Flow) (string, string) {
 	return src, dst
 }
 
-func fmtTimestamp(ts *timestamp.Timestamp) string {
-	t, err := ptypes.Timestamp(ts)
-	if err != nil || t.IsZero() {
+func fmtTimestamp(ts *timestamppb.Timestamp) string {
+	if !ts.IsValid() {
```

Comment:
Very nice find. For other reviewers: `IsValid` checks for nil and invalid timestamps both.